scripts/index.js

```diff
@@ -20,8 +24,23 @@ let curriculumStats = curriculum.getStats();
 //   }
 // }
 
-for (let topicString in curriculumStats) {
-  let topic = curriculumStats[topicString];
-  console.log(topicString, topic)
+/**
+ * Writes Course pages for wiki in `curriculum.wiki` local directory
+ *
+ */
+function writeCoursePages() {
```

Comment:
index is getting a little weird, this should probably live in an updateWiki.js script in the `/scripts` folder.

Reply:
:+1: